src/services/DebtService.sol

```diff
@@ -63,112 +74,88 @@ contract DebtService is AdminService {
         IPool(AAVE_POOL).supply(_bToken, _bAmt, address(this), 0);
     }
 
-    /**
-     * @notice Increases the collateral amount backing this contract's loan.
-     * @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
-     */
+    /// @notice Increases the collateral amount backing this contract's loan.
+    /// @dev This function is only callable by the owner account.
+    /// @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
     function addCollateral(uint256 _cAmt) public payable onlyOwner {
-        // 1. Transfer collateral from owner to this contract
         SafeTransferLib.safeTransferFrom(ERC20(C_TOKEN), msg.sender, address(this), _cAmt);
-
-        // 2. Approve Aave to spend _cAmt of this contract's C_TOKEN
         SafeTransferLib.safeApprove(ERC20(C_TOKEN), AAVE_POOL, _cAmt);
-
-        // 3. Supply collateral to Aave
         IPool(AAVE_POOL).supply(C_TOKEN, _cAmt, address(this), 0);
     }
 
-    /**
-     * @notice Increases the collateral amount for this contract's loan with permit, obviating the need for a separate approve tx.
-     *         This function can only be used for ERC-2612-compliant tokens.
-     * @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
-     * @param _deadline The expiration timestamp of the permit.
-     * @param _v The V parameter of ERC712 signature for the permit.
-     * @param _r The R parameter of ERC712 signature for the permit.
-     * @param _s The S parameter of ERC712 signature for the permit.
-     */
+    /// @notice Increases the collateral amount for this contract's loan with permit, obviating the need for a separate approve tx.
+    ///         This function can only be used for ERC-2612-compliant tokens.
+    /// @dev This function is only callable by the owner account.
+    /// @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
+    /// @param _deadline The expiration timestamp of the permit.
+    /// @param _v The V parameter of ERC712 signature for the permit.
+    /// @param _r The R parameter of ERC712 signature for the permit.
+    /// @param _s The S parameter of ERC712 signature for the permit.
     function addCollateralWithPermit(uint256 _cAmt, uint256 _deadline, uint8 _v, bytes32 _r, bytes32 _s)
         public
         payable
         onlyOwner
     {
-        // 1. Approve with permit
         IERC20Permit(C_TOKEN).permit(msg.sender, address(this), _cAmt, _deadline, _v, _r, _s);
-
-        // 2. Add Collateral
         addCollateral(_cAmt);
     }
 
-    /**
-     * @notice Withdraws collateral token from Aave to specified recipient.
-     * @param _token The address of the collateral token to be withdrawn (C_TOKEN or B_TOKEN).
-     * @param _amt The amount of collateral to be withdrawn (units: C_DECIMALS or B_DECIMALS).
-     * @param _recipient The recipient of the funds.
-     */
+    /// @notice Withdraws collateral token from Aave to specified recipient.
```

Comment:
This is not limited to the "collateral token". Can also withdraw the base token.

Reply:
Addressed with @inheritdoc